ui/nuxeo-document-permissions/nuxeo-popup-permission.js

```diff
@@ -233,7 +233,7 @@ import '../widgets/nuxeo-user-suggestion.js';
         },
         updatingACE: {
           type: Boolean,
-          computed: 'isUpdatingACE(ace)',
+          computed: 'isUpdatingACE()',
```

Comment:
You may want to keep `ace` here, since we want to be aware of observable changes (https://polymer-library.polymer-project.org/3.0/docs/devguide/observers#computed-properties)